cloudbuild-tag.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+- id: Retag
+  name: gcr.io/cloud-builders/docker
+  entrypoint: bash
+  args:
+  - -exc
```

Comment:
`-exc` makes it line ending sensitive? nice!

Reply:
`-exc` is shorthand for `-e -x -c`.